rootfs/etc/nginx/lua/nginx/ngx_conf_balancer.lua

```diff
@@ -0,0 +1,2 @@
+local balancer = require("balancer")
+balancer.balance()
```

Comment:
```suggestion
balancer.balance()
```

Reply:
Can we have the newline at EOF added in snippets?

Reply:
sorry I am not following. What would be the goal of adding a new line here?